job-server/src/spark.jobserver/io/JobSqlDAO.scala

```diff
@@ -93,38 +95,13 @@ class JobSqlDAO(config: Config) extends JobDAO {
       }
     }
 
-    // Create the tables if they don't exist
-    db withSession {
-      implicit session =>
-
-        if (isAllTablesNotExist()) {
-          // All tables don't exist. It's the first time the Job Server runs. So, create all tables.
-          logger.info("Creating JARS, CONFIGS and JOBS tables ...")
-          jars.ddl.create
-          configs.ddl.create
-          jobs.ddl.create
-        } else if (isSomeTablesNotExist()) {
-          // Only some tables exist, not all. It means there is data corruption in the metadata-store.
-          // Exit the Job Server now.
-          throw new RuntimeException("Some tables in metadata-store are missing. Please recover it first.")
-        }
-
-        // If the cases above are not true, it means all tables exit. No need to initialize the database.
-    }
+    // Flyway migration
+    val flyway = new Flyway()
+    flyway.setDataSource(jdbcUrl, jdbcUser, jdbcPassword)
+    flyway.setLocations(migrateLocations)
```

Comment:
This logical needs for tests. It isn't need if test will have reference.conf or something else.